src/app/core/http-services/get-user-by-login.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
```

Comment:
The spec says to create the manager with minimum permissions but the payload is:
```json
{
    "can_manage": "none",
    "can_grant_group_access": true,
    "can_watch_members": true
}
```
I think `can_manage` is correct and the two other props should be false, can you check ?